keadm/cmd/keadm/app/cmd/util/common.go

```diff
@@ -226,9 +226,11 @@ func GetLatestVersion() (string, error) {
 	//Download the tar from repo
 	versionURL := "curl -k " + latestReleaseVersionURL
 	cmd := exec.Command("sh", "-c", versionURL)
+	var stderr bytes.Buffer
+	cmd.Stderr = &stderr
```

Comment:
Could we simpify it as `cmd.Stderr = &bytes.Buffer{}`?

Reply:
if we do,we should also change this line:
`return "", fmt.Errorf("%v:%v", err, stderr.String())`
to
`return "", fmt.Errorf("%v:%v",err,cmd.Stderr.(*bytes.Buffer).String())`

i don't think it's simpified.

Reply:
Could we call Command from [util pkg](https://github.com/kubeedge/kubeedge/blob/master/pkg/util/util.go#L453-L462)?

Reply:
yes, it's a better method.

so do we need to add more error print to util.Commond() like this:
```
//Command executes command and returns output
func Command(name string, arg []string) (string, error) {
	cmd := exec.Command(name, arg...)
	var stderr bytes.Buffer
	cmd.Stderr = &stderr
	ret, err := cmd.Output()
	if err != nil {
		err = fmt.Errorf("%v:%v", err, stderr.String())
		klog.Errorf("exec command failed: %v", err)
		return string(ret), err
	}
	return strings.Trim(string(ret), "\n"), nil
}
```
?

Reply:
Looks good, just add blank after ":".

Reply:
ok, like this : `err = fmt.Errorf("%v:\n%v", err, stderr.String())`;
and the travis-ci tells that `[It] E2E_UPDATE_DEVICE_1`  and `[It] E2E_POD_DEPLOYMENT` get failed ,should i care about them?

Reply:
Sorry, i mean empty space like "%v: %v". 
Some cases maybe failed since little resource, so please rerun test.

Reply:
ok